projects/step-frontend/src/lib/modules/admin/components/my-account/my-account.component.scss

```diff
@@ -35,4 +49,16 @@ step-my-account {
   .actions {
     width: 2rem;
   }
+
+  step-description-hint {
+    display: flex;
+    align-items: center;
+    padding-left: 0.5rem;
+    font-size: 1.4rem;
+  }
+
+  .title-with-hint {
+    display: flex;
+    flex-direction: row;
+  }
```

Comment:
The same could be achieved by adding some styles in title container only
```suggestion
  .title-with-hint {
    display: flex;
    line-height: 2rem
    gap: .5rem;
  }
```

Reply:
"Preferences" and '?' icon are not aligned anymore with the styles above

![image](https://github.com/exense/step-frontend/assets/158443736/bce18e52-7457-4018-a0d4-6f61c06844fe)